src/AppBundle/Membership/MembershipRequestHandler.php

```diff
@@ -57,7 +57,7 @@ public function update(Adherent $adherent, MembershipRequest $membershipRequest)
     {
         $adherent->updateMembership($membershipRequest, $this->addressFactory->createFromAddress($membershipRequest->getAddress()));
 
-        $this->manager->flush();
```

Comment:
Yes it's done here https://github.com/EnMarche/en-marche.fr/blob/master/src/AppBundle/Geocoder/Subscriber/EntityAddressGeocodingSubscriber.php#L40.